FaunaDB.Client/Client/DefaultClientIO.cs

```diff
@@ -117,6 +120,7 @@ async Task<StreamingRequestResult> DoStreamingRequestAsync(string data, IReadOnl
             message.Headers.Authorization = authHeader;
             message.Headers.Add("X-FaunaDB-API-Version", "4");
             message.Headers.Add("X-Fauna-Driver", "csharp");
```

Comment:
Aren't we removing `X-Fauna-Driver` since "c-sharp-*" is now being sent in `X-Runtime-Env`?

Reply:
thanks for this one, I've removed the `X-Fauna-Driver` header